vehicle/mkv/software/bms/bms.c

```diff
@@ -203,21 +157,25 @@ static void state_machine_run(void) {
      */
     uint32_t ot = 0;
     uint32_t ut = 0;
+    int16_t min_temp, max_temp;
+
+    // uint16_t pack_temperature;
+    rc = temperature_task(&ot, &ut, &min_temp, &max_temp);
+    // bms_core.pack_temperature = pack_temperature;
+
+    // if (ut > MAX_EXTRANEOUS_TEMPERATURES) {
```

Comment:
Need to uncomment/figure out what's going on. Might be that we just need to _actually_ calculate the thresholds.